pkg/endpoint/bpf.go

```diff
@@ -137,6 +105,7 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	defer f.Close()
 
 	fw := bufio.NewWriter(f)
+	defer fw.Flush()
```

Comment:
The previous implementation checked the result of this `Flush()`. The new version should do this also.